communities/edinburgh-tech-learners/community-assessment.md

```diff
@@ -0,0 +1,120 @@
+
+There are a lot of online learning resources, of course, but I think (and have heard from others!) that the in-person social aspect is sorely missing in Edinburgh.
+
+In an ideal version of this community, there would be a lot more local community events aimed at this sort of intermediate programmer.
+
+### Accessibility
+> - How accessible are the current resources in your community?
+
+As I wrote above, I don't think there are a lot of resources available _at all_ to people in this community.
+An approach that some members of this community might take is attending events that are aimed at skill levels a bit beyond theirs, and are maybe only marginally related to their interests. This sort of approach is a lot easier for some people than others though — it requires people to have the confidence to step outside their comfort zone, into an unfamiliar environment which is difficult to get an idea of ahead of time. It also demands a lot of effort on the part of those members of this community that participate, because there's not a lot of active encouragement and support for this sort of person.
+
+> - Are meetings held during hours where people may be working?
+
+I think that the meetings that _are_ available are usually held at quite sensible times. However, it's of course impossible to schedule anything at a time that works for everything — especially when targetting an audience of students who may have university during the day and work during evenings or weekends.
+
+In an ideal version of this community, events might be held at varying times to allow everybody to have a chance to participate. The challenges to overcome here will be the limited time available to the organiser(s), and the difficulty that may arise in getting people to regularly attend an event that might be at a different time every week — in particular, I'm given to understand that non-regular event times can be especially difficult for autistic people.
+
+> - Is there childcare available?
+
+No regular tech events of any sort that I know of in Edinburgh provide childcare. This would, of course, be a wonderful thing to provide if possible — though childcare is expensive and very likely to be beyond our means.
+
+### Interests
+
+> - Is your community made up only of computer science students? Are you OK with that?
+
+I'd like the community to be as open as possible to everyone who's interested in tech. I think it would be sensible to aim it at computer science students, because there's a level of common ground there that would be easier to build upon and aiming at students might give us more acccess to the resources we need to provide the services this community needs. However, it's very important to me that as many people are included as possible, even those who aren't studying CS, or aren't students at all.
```

Comment:
Because the flip side of this is that with CS students you have a base expectation of skills. If you have a group made of CS students, you know they understand at least a bit about code, and then it's easier for you to support people with no knowledge of code, because you have a ready supply of welcoming mentors. By being more exclusive initially, it'll enable you to be more successful at inclusion later